### ISSUE TITLE
Add scheduler

### ISSUE THREAD

**Issue by ediboc** [iiconocimiento/aisee#23]
Add to the Trainer functions from  lr_scheduler
